apiary.apib

```diff
@@ -2067,6 +2067,7 @@ Get details for a single invoice.
                                         + excluding
                             + tax (object)
                                 + rate: 0.21 (number)
+                                + id: `e2314517-3cab-4aa9-8471-450e73449042` (string)
```

Comment:
Q: Now that we are providing an ID we also need a `type` right?

Reply:
I think that is not necessary, but if this creates some confusion I can rename it to `rate_id`?

Reply:
But that is our format right? We should keep it consistent.

It's also done for `witheld_tax` so why shouldn't we do it for `tax`?

Reply:
Indeed! Thanks for pointing out.